compair/api/comparison_example.py

```diff
@@ -36,7 +37,9 @@ def post(self, course_uuid, assignment_uuid, comparison_example_uuid):
         course = Course.get_active_by_uuid_or_404(course_uuid)
         assignment = Assignment.get_active_by_uuid_or_404(assignment_uuid)
         comparison_example = ComparisonExample.get_active_by_uuid_or_404(comparison_example_uuid)
-        require(EDIT, comparison_example)
+        require(EDIT, comparison_example,
+            title="Comparison Example Update Failed",
+            message="You do not have permission to update this comparison example since you are not the course's instructor.")
```

Comment:
"Comparison Example Not Updated" / "Your role in this course does not allow you to update comparison examples."